Sources/StreamChatUI/Appearance+Fonts.swift

```diff
@@ -4,8 +4,8 @@
 
 import UIKit.UIFont
 
-public extension _UIConfig {
-    struct Font {
+public extension Appearance {
+    struct Fonts {
         public var caption1 = UIFont.preferredFont(forTextStyle: .caption1)
         public var footnoteBold = UIFont.preferredFont(forTextStyle: .footnote).bold
         public var footnote = UIFont.preferredFont(forTextStyle: .footnote)
```

Comment:
How about making all the peoperties `lazy`?

Reply:
@evsaev lazy var and struct don't go together well :/ but I like the idea. I'm not worried about the performance for fonts but for images I can see it can be very beneficial.

Reply:
Hm, why? 🤔 Isn't it just the matter of `var` everywhere?